Basic/DistributedAuthoritySocialHub/Assets/Editor/Tutorials/Tutorial0_SetupUGS/1-Verify And Setup UGS.asset

```diff
@@ -0,0 +1,122 @@
+%YAML 1.1
+%TAG !u! tag:unity3d.com,2011:
+--- !u!114 &11400000
+MonoBehaviour:
+  m_ObjectHideFlags: 0
+  m_CorrespondingSourceObject: {fileID: 0}
+  m_PrefabInstance: {fileID: 0}
+  m_PrefabAsset: {fileID: 0}
+  m_GameObject: {fileID: 0}
+  m_Enabled: 1
+  m_EditorHideFlags: 0
+  m_Script: {fileID: 11500000, guid: ff771ccdf4150419d9ff4d342b069aae, type: 3}
+  m_Name: 1-Verify And Setup UGS
+  m_EditorClassIdentifier: 
+  Title:
+    m_Untranslated: Configure Unity Gaming Services
+  m_Paragraphs:
+    m_Items:
+    - m_Type: 6
+      Title:
+        m_Untranslated: 
+      Text:
+        m_Untranslated: 
+      m_CodeSample: 
+      m_PostInstructionImage: {fileID: 0}
+      m_Tutorial: {fileID: 0}
+      m_Image: {fileID: 0}
+      m_VideoUrl: 
+      m_Video: {fileID: 0}
+      m_CriteriaCompletion: 0
+      m_Criteria:
+        m_Items: []
+      m_MaskingSettings:
+        m_MaskingEnabled: 1
+        m_UnmaskedViews: []
+      m_Summary: 
+      m_Description: 
+      m_InstructionBoxTitle: 
+      m_InstructionText: 
+      m_TutorialButtonText: 
+    - m_Type: 0
+      Title:
+        m_Untranslated: 
+      Text:
+        m_Untranslated: "The <b>Distributed Authority Social Hub Sample</b> utilizes
+          multiple Unity Cloud services for ease of use and scalability. \n\nIn order
+          to use this sample you must first link your <b>Unity Project</b> to <b>Unity
+          Cloud.</b> This tutorial will walk you through the steps to <b>Link your
+          Existing Project to Unity Cloud.</b>\n\n<b>Note:</b> <i>This process can
```

Comment:
```suggestion
          project to Unity Cloud.</b>\n\n<b>Note:</b> <i>You can also
```

Reply:
Why is this section in bold? Is it the title of a tutorial? It should be in sentence case regardless.